Plugins/Json/src/JsonGeometryConverter.cpp

```diff
@@ -720,6 +714,11 @@ json Acts::JsonGeometryConverter::surfaceMaterialToJson(
 
 json Acts::JsonGeometryConverter::volumeMaterialToJson(
     const Acts::IVolumeMaterial& vMaterial) {
+  auto encodeMaterialParameters =
+      [](const Material::ParametersVector& params) -> std::vector<float> {
+    return {params[0], params[1], params[2], params[3], params[3]};
```

Comment:
Same here.

Reply:
No problem here. I will change this. There is also a bug in that line: `param[4]` is never stored.

Reply:
Technically you don't need `param[4]`, IVolumeMaterial use material and not MaterialProperties so there is no thickness.

Reply:
`param[4]` is the molar density and still part of the `Material` description.

Reply:
Ah yes you are right !

Reply:
See above. I will close this conversation for now.